dbms/tests/integration/test_backup_restore/test.py

```diff
@@ -0,0 +1,78 @@
+import os.path
+import pytest
+
+from helpers.cluster import ClickHouseCluster
+from helpers.test_tools import TSV
+
+
+cluster = ClickHouseCluster(__file__)
+instance = cluster.add_instance('instance')
+q = instance.query
+path_to_data = '/var/lib/clickhouse/'
+
+
+@pytest.fixture(scope="module")
+def started_cluster():
+    try:
+        cluster.start()
+        q('CREATE DATABASE test')
+
+        yield cluster
+
+    finally:
+        cluster.shutdown()
+
+
+def exec_bash(cmd):
+    cmd = '/bin/bash -c "{}"'.format(cmd.replace('"', '\\"'))
+    return instance.exec_in_container(cmd)
+
+
+def copy_backup_to_detached(database, table):
+    fp_increment = os.path.join(path_to_data, 'shadow/increment.txt')
+    increment = exec_bash('cat ' + fp_increment).strip()
+    fp_backup = os.path.join(path_to_data, 'shadow', increment, 'data', database, table)
+    fp_detached = os.path.join(path_to_data, 'data', database, table, 'detached')
+    exec_bash('cp -r {}/* {}/'.format(fp_backup, fp_detached))
+
+
+@pytest.fixture
+def backup_restore(started_cluster):
```

Comment:
Could you think of a name for this fixture that fits in better (and can be used in other tests that I mentioned)?

Reply:
Renamed as you suggested.

Reply:
Hmm, seems you forgot to actually rename it. I would suggest the name `freeze_baseline_table` for this fixture.